shared/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
     "extends": "../tsconfig.json",
-    "include": ["src"],
+    "include": ["src", "../backend/src/common/helpers/generate-salt"],
```

Comment:
I think this might be causing issues?
Because when I remove this from tsconfig and run 'npm run build' inside shared it correctly builds the shared folder.

I might be wrong though.